src/lib/nodes/abstract.node.ts

```diff
@@ -224,9 +268,11 @@ export abstract class Node<C = unknown> {
    * Enables/disables a node.
    */
   toggleNode(active: boolean) {
+    const baseURL = getBaseURL(this)
+
     this.active = active
 
-    nodesStorage.saveActive(this.url, active)
+    nodesStorage.saveActive(baseURL, active)
```

Comment:
Are you sure this will work as expected, considering that the URL passed can now be not only a domain but also an IP address?